src/main/java/seedu/address/logic/LogicManager.java

```diff
@@ -47,19 +49,20 @@ public LogicManager(Model model, Storage storage) {
     public CommandResult execute(String commandText) throws CommandException, ParseException {
         logger.info("----------------[USER COMMAND][" + commandText + "]");
 
-        CommandResult commandResult;
-        Command command = addressBookParser.parseCommand(commandText);
-        commandResult = command.execute(model);
-
+        ArrayList<Command> commands = addressBookParser.parseCommand(commandText);
+        CommandResult[] commandResults = new CommandResult[commands.size()];
+        for (int i = 0; i < commands.size(); i++) {
```

Comment:
Consider changing this to `for (Command command : commands) {//} to simplify`

Reply:
need the index so i cant do this